File: loops/sum.java
import java.util.Scanner;

public class sum {
    public static void main(String args[]) {
        Scanner sc = new Scanner(System.in);
        int sum = 0;
        System.out.println("Enter last number of the series:");
        int range = sc.nextInt();
        int counter = 1;
        while(counter <= range){
            sum = sum + counter;
            counter++; 
        }
        System.out.println("The sum is: "+ sum);

    }
}

File: flowCharts/sum/sum.cpp
#include<iostream>
using namespace std;
int main(){
    int n;
cout<<"Enter no. of natural no. = ";
cin>>n;
int sum = 0,val = 1;
while(val<n){
sum = sum + val;
val = val+1;
}
cout<<"Sum = "<<sum;
}
